simd/unit_tests/include/SIMDTesting_Utilities.hpp

```diff
@@ -106,6 +106,25 @@ class load_element_aligned {
   }
 };
 
+class load_vector_aligned {
+ public:
+  template <class T, class Abi>
+  bool host_load(T const* mem, std::size_t n,
+                 Kokkos::Experimental::simd<T, Abi>& result) const {
+    if (n < result.size()) return false;
+    result.copy_from(mem, Kokkos::Experimental::vector_aligned_tag());
```

Comment:
```suggestion
    result.copy_from(mem, Kokkos::Experimental::vector_aligned);
```
and similarly elsewhere.

Reply:
Replaced all tag uses in tests with `simd_flag_default` and `simd_flag_aligned`.